tests/expr_and_series/list/get_test.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import annotations
+
+import re
+
+import pytest
+
+import narwhals as nw
+from tests.utils import PANDAS_VERSION, Constructor, ConstructorEager, assert_equal_data
+
+data = {"int": [[1, 2], [None, 3], [None]]}
+expected = {"int": [1, None, None]}
```

Comment:
What happens (and should happen) for a null element? not a list with a null
Can we add a test for that?